docs/sources/release-notes/v2-9.md

```diff
@@ -1,31 +1,41 @@
-
--  **Remote rule evaluation**: Rule evaluation can now be handled by queriers to improve speed. [PR #8744](https://github.com/grafana/loki/pull/8744) [PR #8848](https://github.com/grafana/loki/pull/8848)
-
--  **Multi-store Index support**: Loki now supports reading/writing indexes to multiple object stores which enables the use of different storage buckets across periods for storing index. [PR #7754](https://github.com/grafana/loki/pull/7754), [PR #7447](https://github.com/grafana/loki/pull/7447)
-
--  **New volume and volume_range endpoints**: Two new endoints, `index/volume` and `index/volume_range`, have been added to Loki. They return aggregate volume information from the TSDB index for all streams matching a provided stream selector. This feature was introduced via multiple PRs, including [PR #9988](https://github.com/grafana/loki/pull/9988), [PR #9966](https://github.com/grafana/loki/pull/9966), [PR #9833](https://github.com/grafana/loki/pull/9833), [PR #9832](https://github.com/grafana/loki/pull/9832), [PR #9776](https://github.com/grafana/loki/pull/9776), [PR #9762](https://github.com/grafana/loki/pull/9762), [PR #9704](https://github.com/grafana/loki/pull/9704), [PR #10248](https://github.com/grafana/loki/pull/10248), [PR #10099](https://github.com/grafana/loki/pull/10099), [PR #10076](https://github.com/grafana/loki/pull/10076), [PR #10047](https://github.com/grafana/loki/pull/10047) and [PR #10045](https://github.com/grafana/loki/pull/10045)
-
--  **New Storage Client**: Add support for IBM cloud object storage as storage client. [PR #8826](https://github.com/grafana/loki/pull/8826)
-
--  **Block queries by hash**: Queries can now be blocked by a query hash. [PR #8953](https://github.com/grafana/loki/pull/8953)
-
--  **logfmt stage improvements**: logfmt parser now performs non-strict parsing by default which helps scan semi-structured log lines. [PR #9626](https://github.com/grafana/loki/pull/9626)
-
-- **Deprecations**
-  - Legacy index and chunk stores that are not "single store" (such as `tsdb`, `boltdb-shipper`) are deprecated. These storage backends are Cassandra (`cassandra`), DynamoDB (`aws`, `aws-dynamo`), BigTable (`bigtable`, `bigtable-hashed`), GCP (`gcp`, `gcp-columnkey`), and gRPC (`grpc`). See https://grafana.com/docs/loki/latest/storage/ for more information.
-  - The `table-manager` target is deprecated, because it is not used by "single store" implementations.
-  - The `-boltdb.shipper.compactor.*` CLI flags are deprecated in favor of `-compactor.*`.
-  - The `-ingester.unordered-writes` CLI flag is deprecated and will always default to `true` in the next major release.
-  - For the full list of deprecations, see CHANGELOG.md
+---
```

Comment:
Not exactly sure why the whole file changed. I think it is caused by the conversion of the line endings from Windows to Unix, because I saw `^M` characters in the editor at the end of each line.

Reply:
@kavirajk You created the doc initially. Are you aware of any atypical line endings / carriage returns caused by your editor setup?

Reply:
Reverting the file back would add the `^M` as screen is this screenshot

![screenshot_20230914_211905](https://github.com/grafana/loki/assets/281260/051b708c-5f51-4400-9912-cf300338a76d)

Reply:
Reverted the file to its original text format (Mac?).